### ISSUE TITLE
Can't get cookies to work

### ISSUE THREAD

**Issue by dannycb1987** [alexta69/metube#117]
I am trying to download something from Discovery+.

Placed cookies.txt in the right folder (just used same as download folder)

But I keep getting ''Error adding URL: ERROR: [DiscoveryPlus] This video is only available for registered users. You may want to use --cookies.''

What am I missing here?

Installed using portainer. Youtube videos download with no issues. 
![stacked](https://user-images.githubusercontent.com/89314233/153622462-2f6c8261-4af7-4bf5-af21-7c0a19f376ea.png)

**Comment by alexta69** [comment]
The configuration looks all right.. I would suggest you debug it with the yt-dlp command line (using the --cookies parameter to specify the cookies file). If you have it working fine with yt-dlp, and still not working with MeTube, then let me know and perhaps I can help.

**Comment by dannycb1987** [comment]
Thanks! Seems to be something with persmission to read the cookies.txt Any tips?

![image](https://user-images.githubusercontent.com/89314233/153654850-b3e36ab6-2011-4156-861f-1aa527e187aa.png)

**Comment by alexta69** [comment]
Well, at this point it's all down to your specific configuration, which I don't know anything about... I see you're running MeTube with uid 1026, so you have to make sure that this user is able to read this file.

**Comment by dannycb1987** [comment]
Yeah, I've checked everything and it seems fine, but I must be missing something. You helped me further with the commandline tip! Thanks for the help. I'll dive into my permission settings.

**Comment by alexta69** [comment]
Resolved in the thread — it was a file permission issue on cookies.txt. Note that cookies can now also be uploaded via the web UI. Closing.
